lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -1586,6 +1586,13 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
 
   /// Determine if there exists a name mangling for the given type.
   static bool canMangle(TypeBase *Ty) {
+    // TODO: C++ types are not yet supported.
```

Comment:
Could you file a bug with a reproducer and reference the bug here and in all other TODO comments in this PR?

Reply:
Will do. [Here's the link](https://bugs.swift.org/browse/SR-13223).